app/src/main/java/fr/free/nrw/commons/nearby/PlaceRenderer.java

```diff
@@ -112,6 +113,8 @@ public void render() {
             }
         });
 
+        //TODO: Insert onClickListeners for galeryButton and cameraButton
```

Comment:
No, don't do it here 😄 plus, delete the comment

Reply:
Lol sorry, totally forgot about this one!